plugin/manager_windows.go

```diff
@@ -42,16 +42,19 @@ func isExecutableFile(filePath string) (bool, error) {
 // parsePluginName checks if fileName is a valid plugin file name
 // and gets plugin name from it based on spec: https://github.com/notaryproject/specifications/blob/main/specs/plugin-extensibility.md#installation
 func parsePluginName(fileName string) (string, error) {
-	fname := file.TrimFileExtension(fileName)
+	fname := fileName
+	if strings.EqualFold(filepath.Ext(fileName), ".exe") {
```

Comment:
If it returns `false`, should it fail?

Reply:
@shizhMSFT This function is getting the plugin name from file name only. Checking whether the plugin file is executable is done in the next step of the workflow.